serverless.yml

```diff
@@ -0,0 +1,124 @@
+service: app-78523
+
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
```

Comment:
Please specify a region, like `region: eu-west-1`. Not required of course... But. It's better to specify it :) to prevent some issues in the future.